src/screens/home/index.js

```diff
@@ -33,28 +33,40 @@ export default function HomeScreen() {
 			isMounted = false;
 		};
 	}, [isFocused]);
-
 	return (
 		<View style={styles.fileWrap}>
 			<View style={styles.container}>
-				<Gestures
-					style={styles.container}
-					ref={(c) => setGestures(c)}
-					onScaleStart={() => {
-						setDraggable(true);
-					}}
-					onScaleEnd={() => {
-						gestures.reset(() => {});
-						setDraggable(false);
-					}}
-					rotatable={false}
-					draggable={draggable}
-					scalable={zoomable && { min: 1, max: 10 }}>
-					<Image
-						style={styles.preview}
-						source={image}
-					/>
-				</Gestures>
+				{images.length > 1 ? (
+					images.map((image) => (
+						<Image
+							style={{
+								width: `${95 / images.length}%`,
+								height: `${95 / images.length}%`
+							}}
+							key={image.uri}
+							source={{ uri: image.uri }}
+						/>
+					))
+				) : (
+					<Gestures
+						style={styles.container}
+						ref={(c) => setGestures(c)}
+						onScaleStart={() => {
+							setDraggable(true);
+						}}
+						onScaleEnd={() => {
+							gestures.reset(() => {});
+							setDraggable(false);
+						}}
+						rotatable={false}
+						draggable={draggable}
+						scalable={zoomable && { min: 1, max: 10 }}>
+						<Image
+							style={styles.preview}
+							source={images[0]}
+						/>
+					</Gestures>
+				)}
```

Comment:
for readability (and sanity) reasons we should avoid ternaries wherever possible
```suggestion
				{ if (images.length > 1) { return (
					images.map((image) => (
						<Image
							style={{
								width: `${95 / images.length}%`,
								height: `${95 / images.length}%`
							}}
							key={image.uri}
							source={{ uri: image.uri }}
						/>
					))
				) 
				} else return  (
					<Gestures
						style={styles.container}
						ref={(c) => setGestures(c)}
						onScaleStart={() => {
							setDraggable(true);
						}}
						onScaleEnd={() => {
							gestures.reset(() => {});
							setDraggable(false);
						}}
						rotatable={false}
						draggable={draggable}
						scalable={zoomable && { min: 1, max: 10 }}>
						<Image
							style={styles.preview}
							source={images[0]}
						/>
					</Gestures>
				)}
```